src/openmethane_prior/layers/omGFASEmis.py

```diff
@@ -210,7 +217,7 @@ def processEmissions(config: PriorConfig, startDate, endDate, forceUpdate: bool
     cmaqAreas = np.ones(LAT.shape) * cmaqArea  # all grid cells equal area
 
     for i in range(gfasTimes.size):
-        dates.append(startDate + datetime.timedelta(days=i))
+        dates.append(gfasTimes[i])
```

Comment:
This change could be problematic. The previous version iterated based on the provided `startDate` which would presumably align with the `startDate` supplied to other layers. However this didn't appear to work even with the legacy ADS API when running in test scenarios.

This change uses the dates of the data actually returned by the cdsapi call, but there's a risk that these dates are out of alignment from the datasets used by every other layer, if there are bugs in the cdsapi (which I think there are).